docs/running-ops.md

```diff
@@ -0,0 +1,15 @@
+# Running Operations Documentation
+
+## Push a new Docker image to the registry
+
+You must be logged into the `wptdashboard` project in `gcloud`. Be advised
+this is risky as all Jenkins builds pull and use this container.
+
+```sh
+LOCAL_NAME=wptd-testrun-jenkins
```

Comment:
Version number?

Reply:
I think I addressed this in another comment, but this has to have a specific tag in order to be picked up by Jenkins. As an addition we can push versioned images and then promote them to the prod image name after testing.